<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GradientEditorBase</class>
 <widget class="QFrame" name="GradientEditorBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="GradientPreview" name="Preview">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>3000</width>
       <height>70</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="stopPosLabel">
     <property name="text">
      <string>Position:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ScrSpinBox" name="stopPos">
     <property name="suffix">
      <string> %</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="stopColorLabel">
     <property name="text">
      <string>Color:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="ColorCombo" name="stopColor"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="stopOpacityLabel">
     <property name="text">
      <string>Opacity:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="ScrSpinBox" name="stopOpacity">
     <property name="suffix">
      <string> %</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="stopShadeLabel">
     <property name="text">
      <string>Shade:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="ScrSpinBox" name="stopShade">
     <property name="suffix">
      <string> %</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorCombo</class>
   <extends>QComboBox</extends>
   <header>ui/colorcombo.h</header>
  </customwidget>
  <customwidget>
   <class>GradientPreview</class>
   <extends>QFrame</extends>
   <header>ui/gradientpreview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ScrSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">ui/scrspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
